src/expressServer.ts

```diff
@@ -206,9 +217,12 @@ function initializeRoutes(
     next();
   });
 
+  app.use(oapi);
+  app.use("/swagger", oapi.swaggerui);
```

Comment:
I imagine we want a way to secure or even turn off this endpoint as this exposes a lot of interworkings.

Reply:
I disagree. Those inner workings should all be protected on their own, rather than security-through-obscurity (or, this URL isn't obvious, so keeps us around the same level)

Reply:
But! I'll add it as a possible future feature. I can imagine some people using it not agreeing on the above heh.